src/frontend/js/components/Spinner/stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
According to you, does it make sense to name it `index.stories.tsx` ?
From my point of view, with our current syntax it makes sense to have only one story per component folder but some component folder can contains the main component and smaller components only used into the main component.

Does it make sense to add smaller components within storybook or only the main component ?

Reply:
For complex components it may be a good ideas to have multiple story file in order to split stories of the main and sub components. Seems legit to me.